pkg/util/metricmetadataoptions/metricmetadataoptions.go

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/cortexproject/cortex/blob/master/pkg/util/metricmetadataoptions/metricmetadataoptions.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Cortex Authors.
+
+package metricmetadataoptions
```

Comment:
Why is there a need for a whole new package for this? Can't we use just use `ingester_client.MetricsMetadataRequest`?